modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPDefinitionsDisplayContext.java

```diff
@@ -231,6 +216,24 @@ public CreationMenu getChannelsCreationMenu() throws PortalException {
 		).build();
 	}
 
+	public String getCheckedCommerceAccountGroupIds() throws PortalException {
+		return StringUtil.merge(
+			TransformUtil.transformToLongArray(
+				_accountGroupRelLocalService.getAccountGroupRels(
+					CPDefinition.class.getName(), getCPDefinitionId(),
+					QueryUtil.ALL_POS, QueryUtil.ALL_POS, null),
+				AccountGroupRel::getAccountGroupId));
+	}
+
+	public String getCheckedCommerceChannelIds() throws PortalException {
+		return StringUtil.merge(
+			TransformUtil.transformToLongArray(
+				_commerceChannelRelService.getCommerceChannelRels(
+					CPDefinition.class.getName(), getCPDefinitionId(), null,
+					QueryUtil.ALL_POS, QueryUtil.ALL_POS),
+				CommerceChannelRel::getCommerceChannelId));
+	}
+
```

Comment:
moved in a dedicated method for a specific reason?

Reply:
Those methods are also used in [edit_cp_definition_visibility.jsp](https://github.com/liferay-commerce/liferay-portal/pull/5334/commits/4e586dba9fddd721ca3aa259feef0664eaca793c#diff-262e44988122769067adea0dbe0c3536188236d7fb3b00eed5af03ff0585f3e1)